### DISCUSSION TITLE
It is not possible to correctly detect "Overhang"

### DISCUSSION THREAD

**Discussion by Dwinnerlife** [sn4k3/UVtools#920] [Q&A] [answered]
Hello everyone. First of all, thanks for the wonderful software. Without him, everything would be much more difficult. I've been using it for six months and printing on the Sonic Mini 8k and printing very small models on it. They are 2-3 cm high, up to 1 cm^2 in volume, I have a standard print of 20 microns, that is, the pitch is extremely small because of this huge requirement for supports, if there are few of them, then there are strong sagging in the model. As I understand it, Overhang is responsible for this, but I could not get the correct values from it. Help please

In the first screenshot, I marked the problem area. After printing, I noticed that it was heavily blurred, and now I need to reprint it with the addition of support
<img width="620" alt="Screenshot 2024-08-27 123000" src="https://github.com/user-attachments/assets/6b68459d-1934-427b-8770-82a6257258ad">

We are talking about 270, 341, 443, 514, 620 layers, I will attach the ctb file to anyone interested, just tell me how to attach them
<img width="518" alt="image" src="https://github.com/user-attachments/assets/f3a63355-6a5a-4277-98cb-283c368cb05f">
![image](https://github.com/user-attachments/assets/9dab2404-6517-45bc-969e-a8bcb310cccd)

P.s. I don't write well in English and use a translator

**Comment by sn4k3** [comment, marked as answer]
- UVtools does not fix overhangs, you must fix in slicer or use UVtools to add supports where you want.
- A 2mm gap should not be a problem in most cases but depends on resin and exposure
- When printing overhangs you want them to print at full brightness (white / 255) and your bridge starts with a dark grey due the antialiasing or blur, for such small model try to print without antialiasing.

If those small gaps are arc'ing I suggest to add an in between support and other in center of the square.

**Reply by Dwinnerlife** [reply]
Thanks for the answer to remove the blur, it really got better. My resin is extremely liquid, so the requirements for maintenance are high. I guess I'll just have to gain experience) Thank you for the answer
